requirements/requirements_ci.in

```diff
@@ -11,6 +11,6 @@ pytest-cov>=3.0
 pytest>=6.2
 recommonmark>=0.7
 sphinx-rtd-theme>=0.5
-types-dataclasses>=0.6.4
+types-dataclasses>=0.6.4; python_version == '3.6'
```

Comment:
Uhm, I thought we decided that types were not included in the dependencies... This dep may be a left-over which we could also delete if not necessary

Reply:
Dunno why this requirement was still included for python 3.6
It might be possible it is not automatically handled by mypy with that Python version...

Reply:
Done